Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -386,6 +387,11 @@ - (void)_uninstallScreenTimeWebpageController
     [_screenTimeWebpageController removeObserver:self forKeyPath:@"URLIsBlocked" context:&screenTimeWebpageControllerBlockedKVOContext];
     _screenTimeWebpageController = nil;
 }
+
+- (BOOL)_isBlockedByScreenTime
+{
+    return _isBlockedByScreenTime;
+}
```

Comment:
This needs to go under `@implementation WKWebView (WKPrivate)`

Reply:
Also, since the instance variable is conditional on `ENABLE(SCREEN_TIME)`, but the property declaration is not, you probably want to do:

```
(BOOL)_isBlockedByScreenTime
{
#if ENABLE(SCREEN_TIME)
    return _isBlockedByScreenTime;
#else
    return NO;
#endif
}
```